pyproject.toml

```
@@ -1,7 +1,66 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "setuptools_scm>=4.1.0"]
+requires = ["setuptools", "wheel", "setuptools_scm"]
```

`wheel` was in the `setuptools`' docs examples for some while, but was never actually needed. Old setuptools versions would auto-inject the dep, and the modern ones vendor it. It shouldn't be listed, ever.
```suggestion
requires = ["setuptools", "setuptools_scm"]
```

---

Ah, great to know, thanks!